src/plugins/intel_cpu/tests/unit/transformations/split_fc_test.cpp

```diff
@@ -0,0 +1,263 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include <string>
+#include <memory>
+
+#include <openvino/core/model.hpp>
+#include <openvino/opsets/opset1.hpp>
+#include <transformations/cpu_opset/common/op/fully_connected.hpp>
+#include <transformations/cpu_opset/common/pass/split_fc.hpp>
+#include <transformations/init_node_info.hpp>
+#include <transformations/utils/utils.hpp>
+#include <openvino/pass/manager.hpp>
+#include <ov_ops/type_relaxed.hpp>
+
+#include "common_test_utils/ov_test_utils.hpp"
+#include "transformations/rt_info/decompression.hpp"
+
+using namespace testing;
+using namespace ov::intel_cpu;
+
+TEST_F(TransformationTestsF, SplitFCTest) {
+    disable_result_friendly_names_check();
+    disable_rt_info_check();
```

Comment:
> Why do we disable these checks?

- `disable_result_friendly_names_check();` is removed.
- `disable_rt_info_check();` is still keeped because there is no "fused_name_0" string in the transformation. So, this check should be disabled.